### DISCUSSION TITLE
swr not sending data to back

### DISCUSSION THREAD

**Discussion by fuad-hu** [vercel/swr#2642] [Show and tell]
Hi, i am currently using swr version : 2.0.1 , and also use Chakra UI version : 2.0.19 . And integrated tanstack/react-table version 8.9.1. When i get request by api and tru to render table , i didn't get data back, but when i check cache i already see that it is in the cache

**Comment by FaizanMuhammadSE** [comment]
Can you please provide sample example, through sandbox or other online tool
